packages/bpk-component-aria-live/src/BpkAriaLive.js

```diff
@@ -29,17 +29,17 @@ export const POLITENESS_SETTINGS = {
   off: 'off',
   polite: 'polite',
   assertive: 'assertive',
-} as const;
```

Comment:
might need to comment this too since we commented out the rest of the TS code?

Reply:
Good point!